QuantConnect.Polygon/PolygonDataDownloader.cs

```diff
@@ -65,7 +66,7 @@ public IEnumerable<BaseData> Get(DataDownloaderGetParameters parameters)
 
             if (endUtc < startUtc)
             {
-                yield break;
+                return null;
```

Comment:
Could return empty instead of null for this specific case

Reply:
return `return Enumerable.Empty<BaseData>();` instead of `null`.